modules/nf-core/odgi/stats/main.nf

```diff
@@ -0,0 +1,36 @@
+process ODGI_STATS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::odgi=0.8.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/odgi:0.8.0--py39h2add14b_0':
+        'quay.io/biocontainers/odgi:0.8.0--py310hc8f18ef_0' }"
```

Comment:
The container version is different, is it possible to provide the same?

Reply:
There is no py310 version on singularity available, so let me downgrade.